kubernetes/deployment-production.tmpl

```diff
@@ -230,6 +206,8 @@ spec:
               mountPath: "/static-assets"
             - name: panoptes-nginx-config
               mountPath: "/etc/nginx-sites"
+            - name: panoptes-nginx-common.conf
+              mountPath: "/etc/nginx-sites"
```

Comment:
If you're going to keep this in a separate file you'll want to mount it to `/etc/nginx/` instead. `/etc/nginx-sites/*` is included by the main nginx config and including this file there will cause an error.